indico/modules/events/contributions/client/js/index.jsx

```diff
@@ -29,30 +30,38 @@ import PublicationSwitch from './PublicationSwitch';
     if (!editableSubmissionButton) {
       return;
     }
-
     const availableTypes = JSON.parse(editableSubmissionButton.dataset.availableTypes);
     const {eventId, contributionId, contributionCode} = editableSubmissionButton.dataset;
 
-    let responses;
-    try {
-      responses = await Promise.all(
+    const tasks = await Promise.allSettled([
```

Comment:
I did this mainly because `Promise.all` rejects with the reason of the first promise that rejects. However, the second API call to `paperInfoURL` is expected to reject if there is no paper.
But since we require a polyfill, I can go around and not throw.